demo/api/src/console.ts

```diff
@@ -1,26 +1,21 @@
-import { AppModule } from "@src/app.module";
-import { BootstrapConsole } from "nestjs-console";
+import { CommandFactory } from "nest-commander";
 
+import { AppModule } from "./app.module";
 import { createConfig } from "./config/config";
 
 const config = createConfig(process.env);
-const bootstrap = new BootstrapConsole({
-    module: AppModule.forRoot(config),
-    useDecorators: true,
-    contextOptions: {
+
+async function bootstrap() {
+    const appModule = AppModule.forRoot(config);
+
+    // @ts-expect-error CommandFactory doesn't except DynamicModule, only Type<any>
```

Comment:
This can be fixed by removing the DynamicModule type in AppModule, should I change this?

Reply:
Seems like a mess-up on nest-commanders side, though I don't know enough to really have an opinion on it

But there is a typo:

```suggestion
    // @ts-expect-error CommandFactory doesn't accept DynamicModule, only Type<any>
```